package.json

```diff
@@ -51,6 +51,7 @@
     "@swc/core": "~1.7.0",
     "@swc/helpers": "~0.5.2",
     "@trivago/prettier-plugin-sort-imports": "4.3.0",
+    "@tsconfig/node20": "20.1.4",
```

Comment:
Is there another one we should use instead of node20 ? 

https://github.com/tsconfig/bases/tree/main/bases